dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -2898,8 +2895,8 @@ public void AccTypeTest()
             }
         }
 
-        [TestMethod]
-        [TestProperty("TestSuite", "C")]
+        //[TestMethod]
+        //[TestProperty("TestSuite", "C")]
```

Comment:
disabled test ?

Reply:
Yep, I am tracking this in [this issue](https://github.com/microsoft/microsoft-ui-xaml/issues/1818)